o1vm/src/pickles/main.rs

```diff
@@ -125,19 +122,26 @@ pub fn main() -> ExitCode {
             // FIXME
             let start_iteration = Instant::now();
             debug!("Limit of {DOMAIN_SIZE} reached. We make a proof, verify it (for testing) and start with a new chunk");
-            let _proof: Result<Proof<Vesta>, prover::ProverError> =
-                prover::prove::<
-                    Vesta,
-                    DefaultFqSponge<VestaParameters, PlonkSpongeConstantsKimchi>,
-                    DefaultFrSponge<Fp, PlonkSpongeConstantsKimchi>,
-                    _,
-                >(domain_fp, &srs, curr_proof_inputs, &constraints, &mut rng);
+            let proof = prover::prove::<
+                Vesta,
+                DefaultFqSponge<VestaParameters, PlonkSpongeConstantsKimchi>,
+                DefaultFrSponge<Fp, PlonkSpongeConstantsKimchi>,
+                _,
+            >(domain_fp, &srs, curr_proof_inputs, &constraints, &mut rng)
+            .unwrap();
             // FIXME: check that the proof is correct. This is for testing purposes.
             // Leaving like this for now.
             debug!(
                 "Proof generated in {elapsed} μs",
                 elapsed = start_iteration.elapsed().as_micros()
             );
+            let verif = verifier::verify::<
```

Comment:
Can you rebase on @shimkiv PR to have the o1vm e2e tests and have a small `debug!` to time the verification, please?

Reply:
Rebased, reminding myself to add the `debug!`